packages/transmute-dashboard/src/store/transmute/reducer.js

```diff
@@ -1,31 +1,25 @@
 import * as Constants from './constants';
```

Comment:
This is dangling code from the above deleted stuff. We should either fix it entirely or remove it entirely.